lib/govuk_publishing_components/presenters/breadcrumb_selector.rb

```diff
@@ -68,11 +79,6 @@ def options(navigation)
             step_by_step: true,
```

Comment:
Also, do we need this anymore if we're using `breadcrumb_selector.step_by_step`

Reply:
If you look at the `step_by_step` method, it returns the value matching `:step_by_step` within the options hash. So yes - this key is needed.